pkg/translator/prometheusremotewrite/helper.go

```diff
@@ -271,6 +273,36 @@ func addSingleHistogramDataPoint(pt pmetric.HistogramDataPoint, resource pcommon
 
 	}
 
+	// If the max is unset, it indicates the _max metric point should be omitted
+	if pt.HasMax() {
+		// treat max as a sample in an individual TimeSeries
```

Comment:
I don't think this behavior conforms to the otel spec. https://opentelemetry.io/docs/specs/otel/compatibility/prometheus_and_openmetrics/#histograms-1
@kovrus WDYT?

Reply:
Ah thanks for the spec link. How feasible and/or tedious would it be to request an update to that spec on the conversion? I ask because we're losing this min/max meta right now, but it would be useful to have.

Reply:
In my opinion, you should submit a pr on https://github.com/open-telemetry/opentelemetry-specification. But I'm 100% sure, so I think @kovrus @Aneurysm9 may give you the right answer. @jjtimmons